### DISCUSSION TITLE
Introduce myself

### DISCUSSION THREAD

**Discussion by Manish4Kumar** [community/community#47040] [GitHub Education]
I'm Manish Kumar, an undergrad pursuing my B. Tech in Computer Science and Engineering (CSE) at AEC under MAKAUT University, Kolkata.
My main field of interest is Web-Development and Software development. My goal is to get a job in the Software field after graduating. I enjoy meeting people and working with them in a team environment.
Actively looking for internships and willing to connect and collaborate on projects. 
SKILLS -
Languages: C/C++, Python, Java, PHP, HTML, CSS, JavaScript
Database: MySQL
Version Control: Git, GitHub
Other: Shell Scripting, Data Structure, and Algorithms, Problem Solving, Research, MS Office, Google Workspace, Canva
Familiar: Cloud Computing (Google Cloud), AI, ML, and Data Science

**Comment by Lycanchic** [comment]
Welcome and Good Luck with your studies!!
